style.css

```diff
@@ -43,7 +43,7 @@
 
     /* font-family */
     --font-source: 'Source Serif 4', serif; 
-    --font-dm: 'DM Serif Display', serif;
+    --font-dm: 'DM Serif Display', s`erif;
```

Comment:
Should this be serif?

Reply:
oops I will cancel this pull request and send the update one